packages/strapi-plugin-content-type-builder/admin/src/containers/ModelPage/tests/index.test.js

```diff
@@ -159,7 +170,7 @@ describe('<ModelPage />', () => {
 
       expect(redirect.length).toEqual(1);
     });
-    it('should display the EmptyAttributeBlock if the model\'s attributes are empty', () => {
+    it("should display the EmptyAttributeBlock if the model's attributes are empty", () => {
```

Comment:
Same here I would rather have single quotes instead of double ones.

(The prettier and eslint PR is coming soon)